examples/demo-app/esbuild.config.mjs

```diff
@@ -70,7 +68,8 @@ const config = {
   outfile: 'dist/bundle.js',
   bundle: true,
   define: {
-    NODE_ENV
+    'process.env.NODE_ENV': NODE_ENV,
+    'process.env': JSON.stringify(process.env)
```

Comment:
@igorDykhta Do we still need `process.env` ? The changes causes error when `yarn start:local` demo-app on my side:
Uncaught ReferenceError: process is not defined
    at default-settings.js:48:22
    at main.js:30:21
store.js:23 Uncaught ReferenceError: NODE_ENV is not defined
    at store.js:23:1
    at main.js:30:21

Reply:
Thanks, @igorDykhta! I just tested on master with clean build, and it shows the error, which refers to default-settings.js that `process.env` is still used: https://github.com/keplergl/kepler.gl/blob/b4c9e0ec4438195c0e0c68cd6172bc17b73b755a/examples/demo-app/src/constants/default-settings.js#L46-L55

Reply:
I see that we are using dotenv to handle the environment variables now. 
I need to create a .env from the .env.template to avoid the warning.
Should we add it to the README.md so that everyone knows the upcoming change? @igorDykhta